setup.py

```diff
@@ -109,7 +94,7 @@
     ],
     zip_safe = False,
     install_requires = [
-        'jinja2',
+        'jinja2>=2.11.2',
```

Comment:
The current version of [`jinja2` is 3.0.1](https://pypi.org/project/Jinja2/#history).  Is this necessary?